NOMIS/nomis.go

```diff
@@ -234,32 +247,32 @@ func main() {
 				}
 				censusInstances.ReleaseDate = rd.Format("2006-01-02T15:04:05.000Z")
 			}
-
 			if strings.HasPrefix(str, "MetadataText") {
 				if str != "MetadataText0" {
 					example, err = CheckSubString(annotation.Text.(string))
 					if err != nil {
-						log.Event(nil, "failed to get metadatatext", log.ERROR, log.Error(err))
+						log.Event(ctx, "failed to get metadatatext", log.ERROR, log.Error(err))
 						os.Exit(1)
 					}
 				}
 				splitMetaData := strings.Split(str, "MetadataText")
 				txtNumber, _ := strconv.Atoi(splitMetaData[1])
 				if splitMetaData[1] == "" && splitMetaData[1] != "0" {
+					note, title := ReplaceStatDis(example, metaTitleInfo[0])
 					*censusInstances.UsageNotes = append(*censusInstances.UsageNotes, models.UsageNote{
-						Note:  example,
-						Title: metaTitleInfo[0],
+						Note:  note,
+						Title: title,
 					})
-
 				} else if splitMetaData[1] != "0" {
+					note1, title1 := ReplaceStatDis(example, metaTitleInfo[txtNumber+1])
 					*censusInstances.UsageNotes = append(*censusInstances.UsageNotes, models.UsageNote{
-						Note:  example,
-						Title: metaTitleInfo[txtNumber+1],
+						Note:  note1,
```

Comment:
I would argue if you have multiple notes/titles then they should be an array rather than defined as new vars and the name of the var incrementing. Would suggest maybe a struct array of usageNotes.